corehq/messaging/smsbackends/vertex/tests/test_response.py

```diff
@@ -14,11 +19,33 @@ def setUp(self):
         self.queued_sms = QueuedSMS()
 
     def handle_success(self):
-        self.vertex_backend.handle_response(self.queued_sms, TEST_SUCCESSFUL_RESPONSE)
+        self.vertex_backend.handle_response(self.queued_sms, 200, TEST_SUCCESSFUL_RESPONSE)
         self.assertEqual(self.queued_sms.backend_message_id, TEST_SUCCESSFUL_RESPONSE)
         self.assertFalse(self.queued_sms.error)
 
     def handle_failure(self):
```

Comment:
the test method names should start with test_ otherwise i don't think they get picked up by the runner